src/ert/ensemble_evaluator/evaluator_connection_info.py

```diff
@@ -23,6 +23,10 @@ def __init__(
     def dispatch_uri(self) -> str:
         return f"{self.url}/dispatch"
 
+    @property
+    def experiment_ingest_uri(self) -> str:
+        return f"{self.url}/experiment_ingest"
```

Comment:
Now this isn't used anywhere anymore, we can likely remove it.

Reply:
> Now this isn't used anywhere anymore, we can likely remove it.

True, forgot on that one :)